app/upgrades/v8/README.md

```diff
@@ -0,0 +1,4 @@
+# V8 Upgrade
+
+The v8 upgrade is an emergency upgraded coordinated according to osmosis governance proposals [225](https://www.mintscan.io/osmosis/proposals/225), [226](https://www.mintscan.io/osmosis/proposals/226). And thus by implication of 225, incentive proposals [222](https://www.mintscan.io/osmosis/proposals/222), [223](https://www.mintscan.io/osmosis/proposals/223), and [224](https://www.mintscan.io/osmosis/proposals/224).
```

Comment:
If someone has capacity, please expand on this README, detailing where to look for the code for 222, 223, 224 changes (like link to the line nums or just the file)

Also explain in maybe a bit more detail the steps for 226 outlined here: https://github.com/osmosis-labs/osmosis/pull/1481/files#diff-415d593d61c31665178b2fd7ba9093bb0a93a5e6e7999962a435463767be9667R51-R57

(copying for ease) 
```

	// Steps for unpooling
        // -1) Validate unpooling pool ID
        // 0) Get all locks for the user to that pool, and iterate over each one.
	// 1) If the lock is superfluid delegated, superfluid undelegate
	// 2) Break underlying lock. This will clear any metadata if things are superfluid unbonding
	// 3) Get duration from {} (Consider if we can handle complexity for already unbonding Locks)
	// 4) ExitPool with these unlocked LP shares
	// 5) Make 1 new lock for every asset in collateral. Many code paths need this assumption to hold
	// 6) Make new lock begin unlocking
	```

Reply:
Done in [d58a101](https://github.com/osmosis-labs/osmosis/pull/1481/commits/d58a1016d82b557d8439ea7281a22cfa1ba85c6b)

Reply:
awesome job!